File: src/main/java/com/gmail/borlandlp/minigamesdtools/arena/scenario/ScenarioChainCreatorHub.java
package com.gmail.borlandlp.minigamesdtools.arena.scenario;

import com.gmail.borlandlp.minigamesdtools.creator.AbstractDataProvider;
import com.gmail.borlandlp.minigamesdtools.creator.CreatorHub;

public class ScenarioChainCreatorHub extends CreatorHub {
    public ScenarioChainController createChain(String ID, AbstractDataProvider dataProvider) throws Exception {
        return (ScenarioChainController) this.create(ID, dataProvider);
    }
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/arena/scenario/ScenarioChainCreatorHub.kt
package com.gmail.borlandlp.minigamesdtools.arena.scenario

import com.gmail.borlandlp.minigamesdtools.creator.AbstractDataProvider
import com.gmail.borlandlp.minigamesdtools.creator.CreatorHub

class ScenarioChainCreatorHub : CreatorHub() {
    @Throws(Exception::class)
    override fun create(itemID: String, dataProvider: AbstractDataProvider): ScenarioChainController {
        return super.create(itemID, dataProvider) as ScenarioChainController
    }
}